system_tests/tests/ccd100.py

```diff
@@ -57,8 +57,11 @@ def test_GIVEN_setpoint_set_WHEN_readback_THEN_readback_is_same_as_setpoint(self
 
     @skip_if_recsim("Can not test disconnection in rec sim")
     def test_GIVEN_device_not_connected_WHEN_get_status_THEN_alarm(self):
-        self._lewis.backdoor_set_on_device('connected', False)
-        self.ca.assert_that_pv_alarm_is('READING', ChannelAccess.Alarms.INVALID)
+        self.ca.assert_that_pv_alarm_is('READING', ChannelAccess.Alarms.NONE)
+        with self._lewis.backdoor_simulate_disconnected_device():
+            self.ca.assert_that_pv_alarm_is('READING', ChannelAccess.Alarms.INVALID)
+        # Assert alarms clear on reconnection
+        self.ca.assert_that_pv_alarm_is('READING', ChannelAccess.Alarms.NONE)
```

Comment:
How do you feel about placing this into a separate private method or new test entirely?